src/wwwroot/template-editor/adapterService.js

```diff
@@ -492,246 +492,15 @@
-          if (input.tagName === "SELECT" && input.hasAttribute("multiple")) {	
-            _ref3 = input.options;	
-            for (_l = 0, _len3 = _ref3.length; _l < _len3; _l++) {	
-              option = _ref3[_l];	
-              if (option.selected) {	
-                formData.append(inputName, option.value);	
-              }	
-            }	
-          } else if (!inputType || ((_ref4 = inputType.toLowerCase()) !== "checkbox" && _ref4 !== "radio") || input.checked) {	
-            formData.append(inputName, input.value);	
-          }	
-        }	
-      }	
-      for (i = _m = 0, _ref5 = files.length - 1; 0 <= _ref5 ? _m <= _ref5 : _m >= _ref5; i = 0 <= _ref5 ? ++_m : --_m) {	
-        formData.append(this._getParamName(i), files[i], files[i].name);	
-      }	
-      return this.submitRequest(xhr, formData, files);	
-    };
-
-    /**
-      * Private function used by overrideUploadFiles
-      */
-    function resolveOption() {
-      var __slice = [].slice;
-      var args, option;
-      option = arguments[0], args = 2 <= arguments.length ? __slice.call(arguments, 1) : [];
-      if (typeof option === 'function') {
-        return option.apply(this, args);
-      }
-      return option;
-    };
-
-    /**
-     * Double of XMLHttpRequest for prototyping
-     */
-    function XMLHttpRequestDouble() {
-      var self = {
-        status: 201,
-        readyState: 0,
-        responseText: '',
-        open: open,
-        send: send,
-        setRequestHeader: setRequestHeader,
-        getResponseHeader: getResponseHeader
-      };
-
-      return self;
-
-      function open(method, url) {
-        traceAdapterCall(self.open, arguments);
-      };
-
-      function send(data) {
-        traceAdapterCall(self.send, arguments);
-        setTimeout(function () {
-          var e = {
-            loaded: 20,
-            total: 100
-          };
-          self.onprogress(e);
-        }, 1000);
-
-        setTimeout(function () {
-          var e = {
-            loaded: 60,
-            total: 100
-          };
-          self.onprogress(e);
-        }, 4000);
-
-        setTimeout(function () {
-          var e = {
-            loaded: 60,
-            total: 100
-          };
-          self.readyState = 4; // Request Completed
-
-          //self.onprogress(e);                  
-          //self.onload(e);
-
-          self.status = 408; //Request Timeout
-          self.onerror(e);
-        }, 6000);
-      };
-
-      function setRequestHeader(key, value) {
-        traceAdapterCall(self.setRequestHeader, arguments);
-      };
-
-      function getResponseHeader(key) {
-        traceAdapterCall(self.getResponseHeader, arguments);
-        return '';
+    function dropZoneConfig() {
+      var response = {
+        url : RELAY_CONFIG.baseUrl + '/accounts/' + loginSession.accountName + '/files',
+        headers: {
+          'Authorization': 'Bearer '+ apiToken
+        }
       };
-    };
+      return response;
```

Comment:
`response` does not look like a good name, what do you think about `config` or simply return an anonymous object?